opencti-platform/opencti-front/src/private/components/settings/users/User.tsx

```diff
@@ -670,7 +670,7 @@ const User: FunctionComponent<UserProps> = ({ data, refetch }) => {
         </Grid>
         <Triggers recipientId={user.id} filterKey="authorized_members.id" />
         <Grid item xs={6} style={{ marginTop: 10 }}>
-          <Typography variant="h4" gutterBottom={true}>
+          <Typography variant="h4" gutterBottom={true} style={{ paddingBottom: '22px' }}>
```

Comment:
Looking closely, 21px seems the right value :)

```suggestion
          <Typography variant="h4" gutterBottom={true} style={{ paddingBottom: '21px' }}>
```